fw-child/acf-json/group_674d41008a9cb.json

```diff
@@ -0,0 +1,75 @@
+{
+    "key": "group_674d41008a9cb",
+    "title": "Variable Meta",
+    "fields": [
+        {
+            "key": "field_674d4100b944a",
+            "label": "Variable ID",
+            "name": "var_name",
+            "aria-label": "",
+            "type": "text",
+            "instructions": "",
+            "required": 0,
+            "conditional_logic": 0,
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "default_value": "",
+            "maxlength": "",
+            "allow_in_bindings": 0,
+            "placeholder": "",
+            "prepend": "",
+            "append": ""
+        },
+        {
+            "key": "field_674d4164b944b",
+            "label": "Related Training",
+            "name": "related_training",
+            "aria-label": "",
+            "type": "post_object",
```

Comment:
I suggest changing this to Relationship type, where we can define the min of the post as 1 and max as 5. This also allows for adding filters (check only Search and Post Type). This is good for editor UX.